cart/cart.css

```diff
@@ -0,0 +1,45 @@
+@import url(/main.css);
+@import url(/responsive.css);
+
+hr {
+  border: 0;
+  border-bottom: 1px solid var(--black);
+}
+#quantity-input {
```

Comment:
using id as a selector. you can switch to class for consistency

Reply:
Ok